advanced-topics/Logging-Loggly/src/PrismSample/App.xaml.cs

```diff
@@ -43,7 +44,7 @@ protected override void RegisterTypes(IContainerRegistry containerRegistry)
 
             // TODO: Be sure to get your Loggly Consumer Token. Paste it here to avoid typing it...
```

Comment:
remove this

Reply:
ignore this one, it's not the contoso sample I thought we were dealing with.

Reply:
Removed.

Reply:
Didn't see your comment (it wasn't showing at all) before posting my last comment in the conversation. Sorry about that.
I did end up removing it after I looked over Prism.Plugins.Logging.Loggly and determined it was probably best to register a new instance of LogglyOptions. It looks like you merged but didn't get those changes though.